code/datums/emotes.dm

```diff
@@ -157,3 +157,40 @@
 		var/mob/living/L = user
 		if(HAS_TRAIT(L, TRAIT_EMOTEMUTE))
 			return FALSE
+
+/datum/emote/coders_screech //This is a shorthand to allow for custom emotes on our side while ignoring any player only checks that come with "me"
+	key = "code"
+
+/datum/emote/coders_screech/run_emote(mob/user, params, type_override, intentional = FALSE, hardcoded = FALSE) //Just override the song and dance
```

Comment:
Why a `hardcoded` parameter if you are already making a new emote type?
You can just redefine the checks there.